cli/sshkey_list.go

```
@@ -49,6 +49,11 @@ func runSSHKeyList(cli *CLI, cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if outOpts.IsSet("json") {
+		describeJSON(sshKeys, false)
```

Why not `return describeJSON(sshKeys, false)`, i.e. why ignore the error?